app/settings/account/-components/connected-emails/component.ts

```diff
@@ -89,7 +91,8 @@ export default class ConnectedEmails extends Component {
 
     @task({ withTestWaiter: true })
     updatePrimaryEmail = task(function *(this: ConnectedEmails, email: UserEmail) {
-        const errorMessage = this.intl.t('settings.account.connected_emails.update_fail');
+        const errorMessage = this.intl.t('settings.account.connected_emails.update_fail',
+            { supportEmail: config.support.supportEmail });
```

Comment:
you will need to add `htmlSafe: true` to the options object here to enable rendering the html in the translation.

Reply:
Ok, adding now. Thank you Fabrice.